functions/Copy-DbaQueryStoreConfig.ps1

```diff
@@ -91,11 +91,11 @@ Copy the Query Store configuration of the AdventureWorks database in the ServerA
 		catch
 		{
 			Write-Warning "Can't connect to $Source."
-			continue
+			break
 		}
 		
 		# Grab the Query Store configuration from the SourceDatabase through the Get-DbaQueryStoreConfig function
-		$SourceQSConfig = Get-DbaQueryStoreConfig -SqlServer $sourceserver -Databases $SourceDatabase
+		$SourceQSConfig = Get-DbaQueryStoreConfig -SqlServer $Source -Databases $SourceDatabase
```

Comment:
$sourceserver is the full SMO object which has already authenticated, I'll change this part back unless you see an isssue